libs/SalesforceReact/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ plugins {
 
 dependencies {
     api(project(":libs:MobileSync"))
-    api("com.facebook.react:react-native:0.70.14")
+    api("com.facebook.react:react-android:0.72.7")
```

Comment:
This is a specific dependency change from the React Native upgrade helper, as shown in the template repository apps.  However, note here a specific version is used since we seem to be unable to apply the React Native Gradle plugin to our library module.  If anyone knows of React Native documentation specific to integrating it with non-app modules, let me know since I wasn't able to find it.